bnd.bnd

```diff
@@ -0,0 +1 @@
+Private-Package: org.apache.sling.resource.inventory.impl
```

Comment:
This might not be needed, can you check if the manifest stays the same without it?

Reply:
you are right, it's not needed anymore, verified the results after removing bnd file. 

```
REMOVED    HEADER     Bundle-Version:1.0.8
 ADDED      HEADER     Bundle-Version:1.0.9
 ADDED      HEADER     Implementation-Title:Apache Sling Inventory Printer Based on Resources
 ADDED      HEADER     Implementation-Vendor:The Apache Software Foundation
 ADDED      HEADER     Implementation-Version:1.0.9-SNAPSHOT
 ADDED      HEADER     Private-Package:org.apache.sling.resource.inventory.impl